addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1492,7 +1492,7 @@ const Wysiwyg = Widget.extend({
             }
         };
         if (!this.options.snippets) {
-            $toolbar.find('#justify, #table, #media-insert').remove();
```

Comment:
There is more dead code in other places I think. I seem to remember there is also some code to remove the table option in case `this.options.snippets` is `true`, that is in `mass_mailing` and in `website`.

Reply:
Could not find those...
Note that the remove line

Reply:
Could not find those...
But note that line addons/web_editor/static/src/js/editor/snippets.editor.js:4051 did that (see comment [above](https://github.com/odoo/odoo/pull/111131/commits/a7148cc9652fee6f84dc7735e63696113c835d4b#r1090601572)).